code/src/Music.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import data from './data.json'
+
+
+const oldList = data.albums.items 
+
+export const albumNew = []
+
+const ArtistsName = (oldList) => {
+
+    const newList  = []
+
+    oldList.forEach(item => {
+        newList.push({name: item.name, link: item.external_urls.spotify})
+    })
+
+    //let artistWord = newList.toString()
+    //artistWord.replace(",", ", ")
+    return  newList 
+
+}
+
+oldList.forEach((item => {
+    const num = item.artists.length
+    if (num === 1){
+        albumNew.push({
+            name: item.name,
+            image: item.images[0].url,
+            linkAlbum: item.external_urls.spotify,
+            artist: ArtistsName(item.artists),
+            artist1: item.artists[0].name,
+            link1: item.artists[0].external_urls.spotify,
+            artist2: '',
+            link2: '#',
+            artist3: '',
+            link3:'#'
+        })
+    
+    
+    }
```

Comment:
As some of the data here is the same you don't need to repeat it everywhere. You can do something like this to make it easier
```
albumNew.push({
  name: item.name,
  image: item.images[0].url,
  linkAlbum: item.external_urls.spotify,
  artist: ArtistsName(item.artists),
})

if(num < 3) {
  albumNew.push({
    artist: item.artists[num + 1].name,
    link: item.artists[num + 1].external_urls.spotify,
  })
} else {
  albumNew.push({
    artist: '',
    link: '',
  })
}
```